data_steward/constants/cdr_cleaner/clean_cdr.py

```diff
@@ -7,6 +7,9 @@
 DEID_BASE = 'deid_base'
 DEID_CLEAN = 'deid_clean'
 FITBIT = 'fitbit'
+CONTROLLED_DEID = 'controlled_deid'
```

Comment:
Since this will be the starting of the controlled tier. I suggest we maintain similar names between the stage names and variable names. if we choose to use **_controlled_tier_*_**  or the **_controlled_*_**. we should aim to use any one of these conventions which will avoid a lot of confusion in the future.

Reply:
I chose to name them all controlled_tier_* to be as explicit as possible!